apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generated by `createschema miniapp.B2BAppAccessRightSet 'app:Relationship:B2BApp:CASCADE;'`
+ */
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2BAppAccessRightSet')
+
+
+const B2BAppAccessRightSet = new GQLListSchema('B2BAppAccessRightSet', {
+    schemaDoc: 'A set of access rights for a service user integrated with B2BApp.\n' +
+        ' The user can create a set of necessary rights for B2BApp to work.\n' +
+        ' Then it is:\n' +
+        '   - approved (for the corresponding B2BAppAccessRight the link in the "accessRightSet" field to the desired set is specified)\n' +
+        '   - or rejected.',
+    fields: {
```

Comment:
How set records will be distinguished from each other from user point of view.
There is no "name" field or something like this…

Reply:
For now, we have decided to prevent users from changing this model. Only supports and admins can do this. We also added a restriction so that only one set of rights can be created for one B2BApp. This is done for the reason that the functionality with the creation of several sets of rights on behalf of the service user is currently not a priority and has not been carefully developed. 
For the same reason, right now we don't need a "name" field or anything similar.

Reply:
Description updated